pyfixest/report/visualize.py

```diff
@@ -435,50 +439,57 @@ def _coefplot_matplotlib(
 
     Returns
     -------
-    object
-        A seaborn Plot object.
-
-    See Also
-    --------
-    - https://seaborn.pydata.org/tutorial/objects_interface.html
+    matplotlib.figure.Figure
+        A matplotlib Figure object.
     """
     if labels is not None:
         interactionSymbol = " x "
         df["Coefficient"] = df["Coefficient"].apply(
             lambda x: _relabel_expvar(x, labels, interactionSymbol)
         )
+
     yintercept = yintercept if yintercept is not None else 0
-    ub, lb = alpha / 2, 1 - alpha / 2
+    critval = norm.ppf(1 - alpha / 2)
```

Comment:
Instead of computing the `critval` here, could we use the CIs returned via the `tidy()` method directly? Or would that not work with `ax.errorbar`?

Reply:
ax.errorbar expects an offset [so it computes UB and LB automatically under the hood], which is why I went with this. Alternatively, one can also pass a list as in the example above `
    yerr= [restable["Estimate"] - uniform_res["unif_25"], uniform_res["unif_975"] - restable["Estimate"]]`, which is a bit clunky.